operator/internal/common/extensions/mapkubeapis.go

```diff
@@ -68,7 +93,18 @@ func (r *mapKubeAPIsExtensionRun) Execute() error {
 		KubeConfig:       r.config.KubeConfig,
 	}
 
-	if err := mapkubeapisV3.MapReleaseWithUnSupportedAPIs(mapOptions); err != nil {
+	var extra []*mapping.Mapping
+	for _, dubiousGVK := range r.config.DubiousAPIs {
+		if _, err := r.config.RESTMapper.RESTMapping(dubiousGVK.GroupKind(), dubiousGVK.Version); meta.IsNoMatchError(err) {
```

Comment:
Perhaps on other kinds of errors we should log a warning for better debuggability... 🤔